Sources/MapboxNavigation/MapView.swift

```diff
@@ -4,27 +4,38 @@ import MapboxMaps
 import MapboxDirections
 import MapboxCoreNavigation
 
+
+private let trafficTileSetIdentifiers = Set([
+    "mapbox.mapbox-traffic-v1",
+    "mapbox.mapbox-traffic-v2-beta"
+])
+
+private let incidentsTileSetIdentifiers = Set([
+    "mapbox.mapbox-incidents-v1",
+    "mapbox.mapbox-incidents-v2-beta"
+])
+
 /**
  An extension on `MapView` that allows for toggling traffic on a map style that contains a [Mapbox Traffic source](https://docs.mapbox.com/vector-tiles/mapbox-traffic-v1/).
  */
 extension MapView {
     
     /**
-     Returns a list of tile set identifiers for specific `sourceIdentifier`.
+     Returns a set of tile set identifiers for specific `sourceIdentifier`.
      
      - parameter sourceIdentifier: Identifier of the source, which will be searched for in current style of the `MapView`.
-     - returns: List of tile set identifiers.
+     - returns: Set of tile set identifiers.
      */
-    func tileSetIdentifiers(_ sourceIdentifier: String) -> [String] {
+    func tileSetIdentifiersSet(_ sourceIdentifier: String) -> Set<String> {
```

Comment:
I do not think that we usually mention returned type in method signature. Shall we rename it to `tileSetIdentifiers` even though if we have two methods, which return array and set?

Reply:
Yes, thanks for the suggestion, I'll rename it to `tileSetIdentifiers`